chandra_aca/drift.py

```diff
@@ -17,28 +17,12 @@
 from astropy.table import Table
 from astropy.utils.data import download_file
 from Chandra.Time import DateTime
+from ska_helpers.paths import aca_drift_model_path
 from ska_helpers.utils import LazyDict
 
-# Name of environment variable to override default root for data files
-DATA_ROOT_ENV_VAR = "THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW"
```

Comment:
For me, this is now not passing the drift tests which have that monkeypatch code to set DATA_ROOT_ENV_VAR.

Reply:
fixed